src/osu_map_parser.c

```diff
@@ -561,8 +557,8 @@ OsuIntegerVector	OsuMap_getIntegerVector(char *str, char *err_buffer, jmp_buf ju
 		free(elems);
 		longjmp(jump_buffer, true);
 	}
-	vector.x = OsuMap_getInteger(elems[0], 0, 0, err_buffer, jump_buffer);
-	vector.y = OsuMap_getInteger(elems[1], 0, 0, err_buffer, jump_buffer);
+	vector.x = (int)OsuMap_getInteger(elems[0], 0, 0, err_buffer, jump_buffer);
+	vector.y = (int)OsuMap_getInteger(elems[1], 0, 0, err_buffer, jump_buffer);
```

Comment:
I think the cleaner fix would be to change the type for the vectors to be longs instead